pkg/gatewayserver/scheduling/clock.go

```diff
@@ -59,10 +59,13 @@ func (c *RolloverClock) SyncTime() (time.Time, bool) {
 // Sync synchronizes the clock with the given concentrator timestamp and the server time.
 func (c *RolloverClock) Sync(timestamp uint32, server time.Time) ConcentratorTime {
 	rollovers := int64(c.absolute/ConcentratorTime(time.Microsecond)) >> 32
-	if passed := int64(timestamp) - int64(c.relative); passed < 0 {
+	// If the new timestamp is lower, it is considered a roll over if it advanced more than 25%.
```

Comment:
Nit; 
```
// If the new timestamp is lower, it is considered a roll over if the difference is more than 25%.
```

Reply:
Also isn't 25% a lot? We tolerate a drift of up to 17.9 mins?

Reply:
> ```
> // If the new timestamp is lower, it is considered a roll over if the difference is more than 25%.
> ```

On a scale of 0-9, going from 8 to 1, is "the difference" 7 or 3? We're specifically testing if the value advanced more than X, so the 3.

Reply:
> Also isn't 25% a lot? We tolerate a drift of up to 17.9 mins?

It's pretty arbitrary but this is what we ended up with choosing in an offline discussion with @htdvisser

Reply:
The idea behind that 25% was that it is also the margin for FCnt rollovers in earlier LoRaWAN versions (MAX_FCNT_GAP).

Reply:
Yeah ok sounds good. A small note would've been useful.